sites/all/modules/reol_widget/src/Controller/SearchController.php

```diff
@@ -43,6 +43,11 @@ public function search() {
         $query .= ' AND ' . implode(' AND ', $facet_array);
       }
 
+      $sort = $this->getSort($parameters);
+      if ($sort = $this->getSort($parameters)) {
+        $options['sort'] = $sort;
+      }
+
       module_load_include('inc', 'opensearch', 'opensearch.client');
       $result = opensearch_do_search($query, $page, $results_per_page, $options);
```

Comment:
This should be changed to use the SAL layer to do searches,

Reply:
See the https://github.com/ding2/ding2/blob/master/docs/search-abstraction-layer.md